sui/src/unit_tests/cli_tests.rs

```diff
@@ -1362,8 +1362,12 @@ async fn test_split_coin() -> Result<(), anyhow::Error> {
 
     let object_refs = context.gateway.get_owned_objects(address)?;
 
-    // Check log output contains all object ids.
-    let coin = object_refs.get(1).unwrap().0;
+    // Get another coin
+    for c in object_refs {
```

Comment:
I think a more idiomatic practice is to not define `coin` as `mut`, but rather re-introduce `coin` here using iterator + `find`.

Reply:
@lxfind yes I tried using `find` but since `get_object` is async, I'd have to do other stuff to get it to work in the closure.